server/container/guice/memory-guice/src/test/java/org/apache/james/MemoryJamesServerTest.java

```diff
@@ -38,7 +38,8 @@
         return new GuiceJamesServer<>(new TypeLiteral<InMemoryId>(){})
                 .combineWith(MemoryJamesServerMain.inMemoryServerModule)
                 .overrideWith(new TestFilesystemModule(temporaryFolder),
-                        new TestJMAPServerModule(GetMessageListMethod.DEFAULT_MAXIMUM_LIMIT));
+                        new TestJMAPServerModule(GetMessageListMethod.DEFAULT_MAXIMUM_LIMIT),
+                        new TestModule());
```

Comment:
I would not use James to demonstrate that : first, it's slow, second, it involve much components, third, you can't @Ignore the test. You could simply write a test class that create an injector with TestModule.